schema/TRANSFER_TO_AFFILIATES.json

```diff
@@ -0,0 +1,430 @@
+{
+    "$schema": "https://json-schema.org/draft-07/schema#",
+    "$id": "https://github.com/fecgov/fecfile-validate/blob/main/schema/TRANSFER_TO_AFFILIATES.json",
+    "version": "8.3.0.1",
+    "title": "FEC Other Committee Contributions",
+    "description": "SCHEDULE B - Contribution to Other Committee (Line 23), Void of Contribution to Other Committee (Line 23)",
+    "type": "object",
+    "required": [
+        "form_type",
+        "filer_committee_id_number",
+        "transaction_type_identifier",
+        "transaction_id",
+        "entity_type",
+        "payee_organization_name",
+        "payee_street_1",
+        "payee_city",
+        "payee_state",
+        "payee_zip",
+        "expenditure_date",
+        "expenditure_amount",
+        "beneficiary_committee_fec_id",
+        "beneficiary_committee_name"
+    ],
+    "properties": {
+        "form_type": {
+            "title": "FORM TYPE",
+            "description": "",
+            "const": "SB22",
+            "examples": [
+                "SB22"
+            ],
+            "fec_spec": {
+                "FIELD_DESCRIPTION": "FORM TYPE",
+                "TYPE": "A/N-8",
+                "REQUIRED": "X (error)",
+                "SAMPLE_DATA": "SB28B",
+                "VALUE_REFERENCE": "[ SB28B ]",
```

Comment:
Non-functional, but the Sample Data and Value Reference are SB28B instead of SB22